chalice/compat.py

```diff
@@ -1,13 +1,27 @@
 import os
 import platform
+import socket
 import six
 
 
 if six.PY3:
     from urllib.parse import urlparse, parse_qs
+
+    def is_broken_pipe_error(error):
+        # type: (Exception) -> bool
+        return isinstance(error, BrokenPipeError)  # noqa
```

Comment:
Are you sure that's the case?  Pylint doesn't honor `# noqa` AFAIK.  That's for pyflakes/flake8.